titus-supplementary-component/task-relocation-springboot/build.gradle

```diff
@@ -14,14 +14,8 @@
  * limitations under the License.
  */
 
-buildscript {
-    dependencies {
-        classpath "org.springframework.boot:spring-boot-gradle-plugin:2.1.2.RELEASE"
-    }
-}
-
 plugins {
-    id 'org.springframework.boot' version '2.1.2.RELEASE'
+    id 'org.springframework.boot' version '2.1.3.RELEASE'
```

Comment:
I noticed this version is duplicated a lot. Is it possible to use a variable in the top level build.gradle?

Reply:
maybe. I noticed variables are not picked up by all gradle blocks. Let me give it a quick try. If it is simple I will make the change.